bdsx/bds/packets.ts

```diff
@@ -44,17 +44,20 @@ export class DisconnectPacket extends Packet {
 
 @nativeClass(null)
 export class ResourcePacksInfoPacket extends Packet {
-    // unknown
+    @nativeField(bool_t)
+    forcedtoacceot:bool_t;
 }
 
 @nativeClass(null)
 export class ResourcePacksStackPacket extends Packet {
-    // unknown
+    @nativeField(bool_t)
+    forcedtoacceot:bool_t;
 }
 
 @nativeClass(null)
 export class ResourcePackClientResponsePacket extends Packet {
-    // unknown
+    @nativeField(bool_t)
+    forcedtoacceot:bool_t;
```

Comment:
It should be a std::set of std::string and a uint8_t for enum of ResourcePackResponse

Reply:
yes but I am still working on it so I'll add it soon.

Reply:
I am not sure if its possible to add an std::set via bdsx